html1.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <title>HTML accessibility: Task 1</title>
+    <style>
+      body {
+        background-color: #fff;
+        color: #333;
+        font: 1em / 1.4 Helvetica Neue, Helvetica, Arial, sans-serif;
+        padding: 1em;
+        margin: 0;
+      }
+
+      * {
+        box-sizing: border-box;
+      }
+
+      .button {
+        color: white;
+        background-color: blue;
+        border-radius: 10px;
+        width: 170px;
+        padding: 10px;
+        text-align: center;
+        margin-top: 5%;
+      }
+
+      .header1 {
+        font: 70px / 1.4 Helvetica Neue, Helvetica, Arial, sans-serif;
+        margin-top: 0%;
+        margin-bottom: 5%;
+      }
+      
+      .header2 {
+        font: 25px / 1.4 Helvetica Neue, Helvetica, Arial, sans-serif;
+      }
+      
+     </style>
+  </head>
+  <body>
+    <header>
+      <h1 class="header1">Need help?</h1>
+    </header>
+     <p>If you have any problems with our products, our  support center can offer you all the help you need, whether you want:</p>
+      <ol>
+        <li> Advice choosing a new product</li>
+        <li> Tech support on an existing product</li>
+        <li> Refund and cancellation assistance</li>
+      </ol>
+    <h2 class="header2">Contact us now</h2>
+      <p>Our help center contains live chat, e-mail addresses, and phone numbers.</p>
+      <div class="button">Find Contact Details</div>
+    <h2 class="header2">Find out answers</h2>
+      <p>Our Forums section contains a large knowledge base of searchable previously asked questions, and you can always ask a new question if you can't find the answer you're looking for.</p>
+      <div class="button">Access Forums</div>
```

Comment:
hi, Im gonna leave a suggestion
```suggestion
<button>Access Forums</button> ...(in styles in remove the class "." and this allows the <button> tag to embrace the styles).  ...also on the other div>